geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/index/s2/S2Index.scala

```diff
@@ -6,7 +6,8 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  ***********************************************************************/
 
-package org.locationtech.geomesa.index.index.s2
+package org.locationtech.geomesa.index.index
+package s2
```

Comment:
Any particular reason to break out the package name like this?

Reply:
it just gets the `index` package object in scope so we don't have to import things :)

Reply:
Yeah, figured it might be something clever like that.